interfaces/builtin/opengl_test.go

```diff
@@ -123,9 +115,20 @@ func (s *OpenglInterfaceSuite) TestUsedSecuritySystems(c *C) {
 	apparmorSpec := &apparmor.Specification{}
 	err := apparmorSpec.AddConnectedPlug(s.iface, s.plug, nil, s.slot, nil)
 	c.Assert(err, IsNil)
-	c.Assert(apparmorSpec.SecurityTags(), DeepEquals, []string{"snap.client-snap.app-accessing-opengl"})
-	aasnippet := apparmorSpec.SnippetForTag("snap.client-snap.app-accessing-opengl")
-	c.Assert(aasnippet, Equals, expectedSnippet)
+	c.Assert(apparmorSpec.SecurityTags(), DeepEquals, []string{"snap.consumer.app"})
```

Comment:
Thanks.  I'll do a minor change in a separated PR (apparmorSpec -> spec) 
Re: split this to separate methods TestAppArmorSpec
I don't fully understand as I renamed the method from TestUsedSecuritySystems -> TestAppArmorSpec in this PR. Mind to take another look?
We'll definitely 
1. add TestUDevSpec method later after we have udev tagging supported for some interfaces 2. split the method like TestUsedSecuritySystems into TestAppArmorSpec and TestSeccompSpec respectively if there're multiple security backends supported for the interface.